core/services/job/models.go

```diff
@@ -168,6 +168,7 @@ type OffchainReportingOracleSpec struct {
 	ContractConfigTrackerSubscribeInterval models.Interval      `toml:"contractConfigTrackerSubscribeInterval" gorm:"default:null"`
 	ContractConfigTrackerPollInterval      models.Interval      `toml:"contractConfigTrackerPollInterval" gorm:"type:bigint;default:null"`
 	ContractConfigConfirmations            uint16               `toml:"contractConfigConfirmations"`
+	EVMChainID                             *utils.Big           `toml:"evmChainID" gorm:"column:evm_chain_id"`
```

Comment:
It does not apply to webhook specs because the initiation of the job is not chain-dependent.

It only needs to be set on specs that initiate in a chain-dependent way.